tests/setup.py

```diff
@@ -237,8 +316,64 @@ def chain_denoms_to_display(chain_denoms):
 # Create a map of chain denom to token data
 chain_denom_to_data_map = create_chain_denom_to_data_map(all_tokens_data)
 
-# Create a map of pool type to pool data
-pool_type_to_denoms = map_pool_type_to_pool_data(all_pools_data)
+# Create two maps:
+# 1. A map of pool type to pool data
+# 2. A map of denom to top liquidity pool
+pool_type_to_denoms, denom_top_liquidity_pool_map = create_pool_data_maps(all_pools_data)
 
 # Create a map of pool ID to pool data
 pool_by_id_map = {pool.get('pool_id'): pool for pool in all_pools_data}
+
+# Listed tokens that have at least one pool with liquidity
+valid_listed_tokens = choose_valid_listed_tokens() 
+
+# One Transmuter token pair [[pool_id, ['denom0', 'denom1']]]
+transmuter_token_pairs = choose_transmuter_pool_tokens_by_liq_asc(1)
+
+# One Astroport token pair [[pool_id, ['denom0', 'denom1']]]
+astroport_token_pair = choose_pcl_pool_tokens_by_liq_asc(1)
+
+def create_token_pairs():
+    """
+    Selects the following groups of tokens:
+    1. Top 5 by-liquidity
+    2. Top 5 by-volume
+    3. Five low liquidity (between 5000 and 10000 USD)
+    4. Five low volume (between 5000 and 10000 USD)
+
+    Then,
+    - Puts them all in a set
+    - Constructs combinations between each.
+
+    Returns combinations in the following format:
+    [['denom0', 'denom1']]
+    """
+
+    # Five top by-liquidity tokens
```

Comment:
Shall we externalize the 5, 5000, 10000 below to constants.py? Or make them as constants in the beginning of the file setup.py, so that they are not like "embedded" in the middle of the code.

Reply:
[24b2354](https://github.com/osmosis-labs/sqs/pull/211/commits/24b2354592fc515e6e1977a4339dff665f057fe3)